router/router.go

```diff
@@ -103,6 +104,14 @@ func New(cfg Config) http.Handler {
 		}
 	}
 
+	if cfg.LegacySearchRedirectsEnabled {
```

Comment:
Should this segment of code be above line 99? I think `/search` will capture `/searchdata` and `/searchpublication` when SearchRoutesEnabled is true

Reply:
Done